upgrade/network/bican_enable.md

```diff
@@ -0,0 +1,504 @@
+
+   ```json
+   {
+     "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git",
+     "commit": "92ce2c9988fa092ad05b40057c3ec81af7b0af97",
+     "name": "csm-1.9.21",
+     "playbook": "enable_chn.yml"
+   }
+   ```
+
+   **Important:** This new layer *must* run after the COS `ncn-final.yml` layers, otherwise the HSN interfaces will not be configured correctly and this playbook will fail.  Typically at the end of the list is okay.
+
+6. (`ncn#`) Update the NCN personalization configuration.
+
+   ```bash
+   cray cfs configurations update ncn-personalization --file ncn-personalization.json --format toml
+   ```
+
+   Example output:
+
+   ```toml
+   lastUpdated = "2022-05-25T09:22:44Z"
+   name = "ncn-personalization"
+   [[layers]]
+   cloneUrl = "https://api-gw-service-nmn.local/vcs/cray/   slingshot-host-software-config-management.git"
+   commit = "f4e2bb7e912c39fc63e87a9284d026a5bebb6314"
+   name = "shs-1.7.3-45-1.0.26"
+   playbook = "shs_mellanox_install.yml"
+
+   [[layers]]
+   cloneUrl = "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git"
+   commit = "92ce2c9988fa092ad05b40057c3ec81af7b0af97"
+   name = "csm-1.9.21"
+   playbook = "site.yml"
+
+   [[layers]]
+   cloneUrl = "https://api-gw-service-nmn.local/vcs/cray/sat-config-management.git"
+   commit = "4e14a37b32b0a3b779b7e5f2e70998dde47edde1"
+   name = "sat-2.3.4"
+   playbook = "sat-ncn.yml"
+
+   [[layers]]
+   cloneUrl = "https://api-gw-service-nmn.local/vcs/cray/cos-config-management.git"
+   commit = "dd2bcbb97e3adbfd604f9aa297fb34baa0dd90f7"
+   name = "cos-integration-2.3.75"
+   playbook = "ncn.yml"
+
+   [[layers]]
+   cloneUrl = "https://api-gw-service-nmn.local/vcs/cray/cos-config-management.git"
+   commit = "dd2bcbb97e3adbfd604f9aa297fb34baa0dd90f7"
+   name = "cos-integration-2.3.75"
+   playbook = "ncn-final.yml"
+
+   [[layers]]
+   cloneUrl = "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git"
+   commit = "92ce2c9988fa092ad05b40057c3ec81af7b0af97"
+   name = "csm-1.9.21"
+   playbook = "enable_chn.yml"
+   ```
+
+7. (`ncn#`) Check that NCN personalization runs and completes successfully on all worker nodes.
+
+   Updating the CFS configuration will cause CFS to schedule the nodes for configuration. Run the following command to verify this has occurred.
+
+   ```bash
+   cray cfs components describe --format toml x3000c0s4b0n0
+   ```
+
+   Example output:
+
+   ```toml
+   configurationStatus = "pending"
+   desiredConfig = "ncn-personalization"
+   enabled = true
+   errorCount = 0
+   id = "x3000c0s4b0n0"
+   state = []
+   
+   [tags]
+   ```
+
+   `configurationStatus` should change from `pending` to `configured` once NCN personalization completes successfully.
+
+For more information on managing NCN personalization, see [Perform NCN Personalization](../../operations/CSM_product_management/Perform_NCN_Personalization.md).
+
+### Migrate CSM Services (MetalLB)
+
+```text
+TODO
```

Comment:
TODO!

Reply:
We've performed the update and it works.  Attempting to have more automation than a series of "vi this and cut-n-paste that" which makes me sad.